dashboard/db/migrate/20220421224748_create_code_review_notes.rb

```diff
@@ -0,0 +1,20 @@
+class CreateCodeReviewNotes < ActiveRecord::Migration[6.0]
+  def change
+    create_table :code_review_notes do |t|
```

Comment:
Looks like you're missing `deleted_at` here

Reply:
Oops, good catch, thanks!